ulog/README.md

```diff
@@ -22,13 +22,12 @@ import "go.uber.org/fx/ulog"
 func main() {
   // Configure logger with configuration preferred by your service
   logConfig := ulog.Configuration{}
-  logBuilder := ulog.Builder().WithConfiguration(&logConfig)
 
-  // Build ulog.Log from logBuilder
-  log := lobBuilder.Build()
+  // Build logger from logConfig object
+  log, err := logConfig.Build(zap.Hooks(ulog.Metrics(svc.metrics)))
```

Comment:
This should be part of service initialization, but not needed. I'll add simple build here.